### DISCUSSION TITLE
Feed Workaround: Access URL for Following Feed

### DISCUSSION THREAD

**Discussion by sinclairzx81** [community/community#37693] [Other Feature Feedback, Questions, & Ideas]
For anyone stuck on this Feed AB test. You should be able to access the usual Following feed at the URL below.

https://github.com/dashboard-feed

**Comment by hmmhmmhm** [comment]
ah..   finally thx so much

**Comment by falsandtru** [comment]
Looks like just a source (fragment) of an old dashboard. Not made for direct viewing.

**Reply by sinclairzx81** [reply]
Yeah, the old dashboard used to load in this fragment. I'm hoping having to visit it directly is a temporary measure.
